build/parameters.go

```diff
@@ -83,4 +83,6 @@ var WhitelistedBlock = buildconstants.WhitelistedBlock // Deprecated: Use buildc
 
 const Finality = policy.ChainFinality // Deprecated: Use policy.ChainFinality instead
 
+// Discussion on this value and a tracking item to document the probablistic impact of various values is in
```

Comment:
Start the godoc by `SafeHeightDistance` to keep to convention. e.g.

```
// SafeHeightDistance is the distance from the latest tipset, i.e. heaviest, that
// is considered to be safe from re-orgs at an increasingly diminishing
// probability.
//
// This is used to determine the safe tipset when using the "safe" tag in
// TipSetSelector or via Eth JSON-RPC APIs. Note that "safe" doesn't guarantee
// finality, but rather a high probability of not being reverted. For guaranteed
// finality, use the "finalized" tag.
//
// This constant is experimental and may change in the future. See:
//   - https://github.com/filecoin-project/go-f3/issues/944
```

Reply:
Thanks.  Addressed in https://github.com/filecoin-project/lotus/pull/13051/commits/7706ee5174ba1b6bf7f02babe755b614a3b89e51 .  Resolving.